server.js

```diff
@@ -69,24 +82,41 @@ app.get('/users/add', (req, res) => {
     firstname, lastname, email, username, password, type,
   };
   const insertUser = 'INSERT INTO users SET ?';
-  // eslint-disable-next-line consistent-return
   con.query(insertUser, body, (err) => {
     if (err) {
       return res.send(err);
     }
-    res.send('successfully added user');
+    return res.send('successfully added user');
   });
 });
 
-/* fake data for backend work */
-app.get('/api/events', (req, res) => {
-  const events = [
-    { id: 1, firstName: 'John', lastName: 'Doe' },
-    { id: 2, firstName: 'Joe', lastName: 'Swanson' },
-    { id: 3, firstName: 'Kanye', lastName: 'West' },
-  ];
+/* adds event to events table */
+app.get('/events/add', (req, res) => {
+  const {
+    teamName, date, time, location, description,
+  } = req.query;
+  const body = {
+    teamName, date, time, location, description,
+  };
+  const insertEvent = 'INSERT INTO events SET ?';
+  con.query(insertEvent, body, (err) => {
+    if (err) {
+      return res.send(err);
+    }
+    return res.send('successfully added event');
+  });
+});
 
-  res.json(events);
+/* selects team names */
+app.get('/teams', (req, res) => {
+  con.query(selectTeamNames, (err, results) => {
+    if (err) {
```

Comment:
Again consider sending an HTTP status code like 500, instead of the error, also maybe consider logging the error for debugging purposes.